src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -11,8 +11,8 @@ import cromwell.engine.db.DataAccess._
 import cromwell.engine.db.ExecutionDatabaseKey
 import cromwell.engine.db.slick._
 import cromwell.engine.workflow.WorkflowActor.{Restart, Start}
+import cromwell.engine.workflow.WorkflowManagerActor.{WorkflowManagerData, WorkflowManagerState}
```

Comment:
I guess because the Object methods are used in the Class? 

Has anyone ever opined on whether its better to have these all the way up here, or as a line in the class definition?